static/app/gettingStartedDocs/android/android.tsx

```diff
@@ -431,7 +431,33 @@ const replayOnboarding: OnboardingConfig<PlatformOptions> = {
       ],
     },
   ],
-  verify: () => [],
+  verify: () => [
+    {
+      type: StepType.VERIFY,
+      description: t(
+        'To verify your Replay setup, trigger an error in your app and watch Sentry capture the event along with a recording of the user interaction.'
```

Comment:
Since we suggest `session_sample_rate = 1` we can just say open the app and try to add error, as an `optional`.

Something like this:

`To verify your Replay setup, open your app and wait for the session replay to be uploaded to Sentry.`

`To see how errors are sessions are connected, try triggering an error by adding the following code:`

`'After clicking the button, wait a few moments, and you\'ll see the error linked to the session recording.'`